web_widget_timepicker/static/src/js/timepicker_widget.js

```diff
@@ -0,0 +1,89 @@
+odoo.define('timepicker.form_widgets', function (require) {

```

Comment:
I could be wrong here, but I believe it is SOP to namespace the JS modules the same as the actual module. `web_wiget_timepicker.form_widgets` or similar